include/OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystem.hpp

```diff
@@ -0,0 +1,186 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// @project        Open Space Toolkit ▸ Astrodynamics
+/// @file           OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystem.hpp
+/// @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+/// @license        Apache License 2.0
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Flight_System_SatelliteSystem__
+#define __OpenSpaceToolkit_Astrodynamics_Flight_System_SatelliteSystem__
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <OpenSpaceToolkit/Astrodynamics/Flight/System.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+#include <OpenSpaceToolkit/Mathematics/Geometry/3D/Objects/Composite.hpp>
+#include <OpenSpaceToolkit/Mathematics/Geometry/3D/Objects/Cuboid.hpp>
+
+#include <OpenSpaceToolkit/Physics/Units/Mass.hpp>
+
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/Integer.hpp>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+namespace ostk
+{
+namespace astro
+{
+namespace flight
+{
+namespace system
+{
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+using ostk::core::types::Real ;
+
+using ostk::math::obj::Matrix3d ;
+using ostk::math::geom::d3::objects::Cuboid ;
+using ostk::math::geom::d3::objects::Composite ;
+using ostk::math::obj::Vector3d ;
```

Comment:
```suggestion
using ostk::math::obj::Vector3d ;
using ostk::math::obj::Matrix3d ;
using ostk::math::geom::d3::objects::Cuboid ;
using ostk::math::geom::d3::objects::Composite ;
```

Reply:
What is the correct input order? For header files as well as "using"

Reply:
fixed